packages/web-config-server/src/apiV1/utils/fetchIndicatorValues/fetchAggregatedAnalyticsByDhisIds.js

```diff
@@ -94,15 +94,17 @@ const performEntityAggregation = async (models, analytics, entityAggregation, hi
 };
 
 const getQueryInput = (query, dataElementIds, organisationUnitIds) => {
+  const dataPeriodType = query.dataPeriodType || query.periodType;
```

Comment:
We actually had the same problem for the normal Indicators. I solved it by puting the `dataPeriodType` in the `data_source` config. I personally think that's a more ideal solution because then we don't have to keep specifying the same `dataPeriodType` in the `dataBuilderConfig` for every visualisation that use the Indicator data source. 

However I think this is fine as this whole file is a hack and we don't really want to use aggregated analytics endpoint for Program Indicators. We want to use our normal analytics event  (`analytics/events/query/`) endpoint, which should resolve the issue. Once https://github.com/beyondessential/tupaia-backlog/issues/2607 is solved, we can immediately remove this file and use our normal analytics event  (`analytics/events/query/`) endpoint. With that, we won't have to specify the period type anymore.

Reply:
Thanks @chris-bes